build_manpages/manpage.py

```diff
@@ -167,11 +165,12 @@ def _format_action(self, action):
 
         return parts
 
-
     def _format_ag_subcommands(self, actions, prog):
         lines = []
 
         for action in actions:
+            if getattr(action, 'help', None) == SUPPRESS:
+                continue
```

Comment:
Same here, test for this would be awesome.

Reply:
done